squidDrone/src/propulsion/little_bee_20_a.cpp

```diff
@@ -0,0 +1,60 @@
+#include "little_bee_20_a.hpp"
+#include "globals.hpp"
+
+namespace propulsion{
+
+  const auto LittleBee20A::SetPulseDuration(int pulse_duration, int repetition_period) noexcept -> types::HalError{
+    types::HalError error_state;
+    const bool pulse_limit_breach = pulse_duration > ONESHOT_125_MAX_PULSE_DURATION_IN_US_ || pulse_duration < ONESHOT_125_MIN_PULSE_DURATION_IN_US_;
```

Comment:
exceeds line length? (80 characters)

Reply:
fixed in [this commit](https://github.com/elheck/Squiddrone/pull/168/commits/04989db9ed3460ed454f9233b555d70d0787455c)